### PR TITLE
Fix flipped

### PR DESCRIPTION
**Pull request by l4u**
* Flip() requires aFlipLeftRight (it would be cool to parameterize this in the future)
* get flipped from props
* use XOR to simplify the code

### CODE REVIEW

File: component_layout_plugin.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def Run( self ):
                 logger.warning("Did not find component {} in PCB design".format(refdes))
                 continue
             
-            flip = props.get('flip', False) # Generally, flip means put on the bottom
+            flip = props.get('flipped', False) # Generally, flip means put on the bottom
```
Comment (mcbridejc) [inline component_layout_plugin.py]: Arguably 'flipped' is a better name for this property, but not sufficiently better to justify breaking backwards compatibility, so I don't see any reason to change this prop name...

File: component_layout_plugin.py [hunk 2]
```diff
@@ -157,10 +157,8 @@ def Run( self ):
                 else:
                     mod.SetPosition(pcbnew.wxPointMM(x0 + x, y0 + y))
             
-            if flip and not mod.IsFlipped():
-                mod.Flip(mod.GetPosition())
-            if not flip and mod.IsFlipped():
-                mod.Flip(mod.GetPosition())
+            if flip ^ (mod.IsFlipped()):
+                mod.Flip(mod.GetPosition(), False)
```
Comment (mcbridejc) [inline component_layout_plugin.py]:
```suggestion
                if v5_compat:
                    mod.Flip(mod.GetPosition())
                else:
                    mod.Flip(mod.GetPosition(), False)
```

The aFlipLeftRight argument is new in v6.

**Review by mcbridejc — CHANGES_REQUESTED** [review]
Ugh. Thanks for catching this. I guess I didn't try to put any components on the bottom while testing with v6...

Just a couple comments below; I'm trying at least for now to maintain compatibility with v5.

File: component_layout_plugin.py (hunk 1 above)
Comment (mcbridejc) [reply component_layout_plugin.py]: Oh I see I put 'flipped' in the README example, which is a mistake on my part.

**Comment by l4u** [comment]
👍 for v5 compatibility

**Pull request merged**